### PR TITLE
List

### PR DESCRIPTION
**Pull request by bossmonkey98**
Lists page added

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand
Good to merge.

File: pages/image/image.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+.responsive-img{
+    margin: 3em;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (t007rushi) [inline pages/image/image.css:6]: can make an utility

**Review by sanjitsarkar — APPROVED** [review]
...

File: pages/image/image.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+<head>
+    <link rel="stylesheet" href="image.css">
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.0/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="../../docs.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apex UI</title>
```
Comment (t007rushi) [inline pages/image/image.html:12]: have title of specific component

**Review by t007rushi — APPROVED** [review]
great work go ahead

File: pages/image/image.html (hunk 2 above)
Comment (bossmonkey98) [reply pages/image/image.html:12]: ok

File: pages/image/image.css (hunk 1 above)
Comment (bossmonkey98) [reply pages/image/image.css:6]: ok

**Pull request merged**